core/src/main/java/de/danoeh/antennapod/core/feed/FeedItemFilter.java

```diff
@@ -67,7 +85,7 @@ public FeedItemFilter(String[] properties) {
      * Run a list of feed items through the filter.
      */
     public List<FeedItem> filter(List<FeedItem> items) {
-        if(mProperties.length == 0) return items;
+        if (mProperties.length == 0) return items;
```

Comment:
Checkstyle does not like this change. Please either keep the line as it is or add braces. The other checkstyle violations about missing braces can be ignored, for now. I can merge it without those tests passing.

Reply:
:+1: i'm using intelliJ to format the code, i guess it's not so good huh

Reply:
Code styles can be different. There is no "right" or "wrong" way to format files. The most important thing is that it is consistent throughout the project. For AntennaPod, we started to use the Google Java code style a few months ago, which does not permit `if` statements without braces. If you remember the [goto fail vulnerability](https://en.wikipedia.org/wiki/Unreachable_code#goto_fail_bug), you know why ;)

To prevent a huge commit that touches every file, only new code needs to comply. The CI only complains about lines that were changed, so if you do not change the line, it will not complain here. I will make an exception in this PR: The new filters you added can stay single-line to be consistent within the file, even if it does not comply with the code style.